news/2021-01-02-mappers-report-january.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "The Mappers' Report: January”
+date: 2021-02-01 08:00:00 +0000
+---
+
+Happy New Year! Okay, it’s been already a month since we moved to 2021, but still. Just like last year, we will be delivering you the news regarding mapping and modding related topics. So let’s dive right into them!
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers’ Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Around the Community](#around-community)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+The ranking criteria continues to change, even in 2021. Here we’ve gathered all changes which happened so far.
+
+### osu!catch
+
+- **[Allow dashes over 250ms before higher-snapped hyperdashes in Platters]:** This change didn’t add or remove any rules or guidelines. It split existing rules in Platter and Rain difficulties into two separate rules for simplification purposes.
+
+## Open Ranking Criteria Proposals 
+
+All modes:
+
+- **[Beatmaps of explicit audio tracks need to have "Explicit Content" setting enabled](https://osu.ppy.sh/community/forums/topics/1218048)**
+
+- **[Adjusting the song cut guideline](https://osu.ppy.sh/community/forums/topics/1231363)**
+
+For osu!:
+
+- **[Let spinners start/end less than 1/16 from the prior/following object](https://osu.ppy.sh/community/forums/topics/1214161)**
+
+For osu!mania:
+
+- **[Remove Guidelines regarding SVs](https://osu.ppy.sh/community/forums/topics/1227918)**
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+### BN Additions
+
+- **osu!:** [rosario wknd(https://osu.ppy.sh/users/6341518) & [Xahlt](https://osu.ppy.sh/users/13466351)
+
+- **osu!catch:** [Dako](https://osu.ppy.sh/users/11081858) & [Myxo](https://osu.ppy.sh/users/2202645)
+
+- **osu!mania:** [guden](https://osu.ppy.sh/users/11626065) & [deta5859](https://osu.ppy.sh/users/15275662)
```

Comment:
Looks like you forgot taiko BN additions

Reply:
Who was the taiko addition? Either i am blind or they didn't show up in the channel i chacked the additions

Reply:
Ulqui https://osu.ppy.sh/users/1263669 and Hivie https://osu.ppy.sh/users/14102976

Reply:
Hmm.. indeed they are missing. Not sure how that could happen. I'll fix it!